test/inductor/test_aot_inductor.py

```diff
@@ -1750,6 +1750,38 @@ def forward(self, x):
         example_inputs = (torch.randn(16, 16, 16, device=self.device),)
         self.check_model(Model(), example_inputs)
 
+    def test_attr_mutation(self):
```

Comment:
I don't know if it totally makes sense for this to live in `AOTInductorTestsTemplate` since it doesn't matter whether we are in ABI-compat mode or not...